dataclients/kubernetes/kube.go

```diff
@@ -670,6 +671,17 @@ func (c *Client) ingressToRoutes(items []*ingressItem) ([]*eskip.Route, error) {
 			annotationPredicate = val
 		}
 
+		// parse routes annotation
+		var extraRoutes []*eskip.Route
+		annotationRoutes := i.Metadata.Annotations[skipperRoutesAnnotationKey]
+		if annotationRoutes != "" {
+			var err error
+			extraRoutes, err = eskip.Parse(annotationRoutes)
+			if err != nil {
+				log.Errorf("failed to parse routes from %s, skipping: %v", skipperRoutesAnnotationKey, err)
```

Comment:
the namespace and the name of the current ingress can be a useful info in the error.